packages/nix/NIX.md

```diff
@@ -0,0 +1,31 @@
+# How to import
+
+To import with flakes use
+```nix
+inputs = {
+  polymc.url = "github:PolyMC/PolyMC";
+};
+
+...
+
+nixpkgs.overlays = [ inputs.polymc.overlay.${system} ]; ## Within configuration.nix
+environment.systemPackages = with pkgs; [ polymc ]; ##
+```
+
+To import without flakes use channels:
+
+```
+sudo -i nix-channel --add https://github.com/PolyMC/PolyMC/archive/master.tar.gz polymc
+sudo -i nix-channel --update polymc
+nix-env -iA polymc
+```
+
+or alternatively you can use
+
+```
+nixpkgs.overlays = [
+  (import (builtins.fetchTarball "https://github.com/lourkeur/PolyMC/archive/develop.tar.gz")).overlay
```

Comment:
```suggestion
  (import (builtins.fetchTarball "https://github.com/PolyMC/PolyMC/archive/develop.tar.gz")).overlay
```
This shouldn't point to my fork. I used it as a working example earlier because the dev branch was further out. Also maybe it should be the master branch?

Reply:
Were providing nightly packages along with stable packages, so I believe we should still target master branch